qa/tasks/ceph.py

```diff
@@ -1214,6 +1214,40 @@ def first_in_ceph_log(pattern, excludes):
             ),
         )
 
+def osd_scrub_bluestore(ctx, config):
+    """
+    Scrub bluestore allocation after we exit making sure allocation file is consistent with ONodes.
+    OSDs must be down for the command to work!
+    """
+
+    self.log('osd_scrub_bluestore started')
```

Comment:
this function uses self a bunch, which isn't defined here. It looks like it's based on code from ceph_manager, but as written this won't work.

When you're testing a change like this you need to also specify your branch as the qa suite branch so that these changes to teuthology tasks are picked up too - e.g. use ```--suite-branch safe_shutdown_v2``` when scheduling the test run

Reply:
I'd suggest keeping this implementation in ceph_manager.py - you could make it a method for one osd,  ```osd_scrub_bluestore(self, osd_id)``` and loop over the osds here in ceph.py. You can get the relevant CephManager instance here via: ```ceph_manager = ctx.managers[config['cluster']]```